packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -189,7 +189,6 @@ const TooltipTrigger = React.forwardRef<TooltipTriggerElement, TooltipTriggerPro
     return (
       <PopperPrimitive.Anchor asChild {...popperScope}>
         <Primitive.button
-          type="button"
```

Comment:
With the now singled-out inconsistency, I wonder if we should leave a comment to replace this to explain why we don't do it here? I can imagine one of us in a few months coming back and adding it back by mistake.

What do you think?